lib/mistral_ai_summary_example/mistralai_summary_page.dart

```diff
@@ -1,13 +1,165 @@
+                summarySettings = newSettings ?? summarySettings;
+              });
+            },
+            icon: const Icon(Icons.settings),
+          ),
+        ],
+      ),
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.all(16),
+          child: SingleChildScrollView(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                TextField(
+                  decoration: const InputDecoration(
+                    labelText: 'Enter text to summarize',
+                  ),
+                  minLines: 1,
+                  maxLines: 10,
+                  onChanged: (value) => summaryInputController.text = value,
+                ),
+                const SizedBox(height: 20),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        summarizeText(summaryInputController.text);
+                      },
+                      child: const Text('Summarize'),
+                    ),
+                    const SizedBox(width: 20),
+                    Column(
+                      children: [
+                        ElevatedButton(
+                          onPressed: () {
+                            summarizeText(summaryTextSample);
+                          },
+                          child: const Text('Summarize sample tex'),
+                        ),
+                        const SizedBox(height: 20),
+                        TextButton(
+                          onPressed: () => showDialog<String>(
+                            context: context,
+                            builder: (context) =>
+                                const SampleTextDialogButton(),
+                          ),
+                          child: const Text('See sample text'),
+                        ),
+                      ],
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 20),
+                const Text('Summarized Text:'),
+                const SizedBox(height: 10),
+                SelectableText(summaryResult),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class SampleTextDialogButton extends StatelessWidget {
+  const SampleTextDialogButton({super.key});
+
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(
-        child: Text('Summary page'),
+    return Dialog(
+      child: Padding(
+        padding: const EdgeInsets.all(8),
+        child: Column(
+          children: [
+            TextButton(
+              onPressed: () {
+                Navigator.pop(context);
+              },
+              child: const Text('Close'),
+            ),
```

Comment:
I think this button should be at the bottom of the screen.

Reply:
Resolved in [6175491](https://github.com/nomtek/flutter_ai_examples/pull/3/commits/6175491ca6f1db066119f6262f5742a40e440234)